packages/components/src/internal/components/picklist/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const PRIVATE_PICKLIST_CATEGORY = 'PrivatePicklist';
+export const PUBLIC_PICKLIST_CATEGORY = 'PublicPicklist';
```

Comment:
Why not make these an enum?

Reply:
Yes, they definitely should be. I was just focused on moving things around in this set of PRs. Trying not to make any actual changes to the code.